library/src/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ if( BUILD_WITH_TENSILE )
   endif( )
 
   # HACK: We include the config file directly because find_package(Tensile) is broken
-  include(${VIRTUALENV_HOME_DIR}/cmake/TensileConfig.cmake)
+  include( ${Tensile_TensileConfig} )
```

Comment:
No need to change this as tensile should always be installed in the virtualenv.

Reply:
@pfultz2  - I'm not familiar with virtualenv and my primitive attempts to implement the requested change were unsuccessful.    Not sure what the -e switch does but the install location appears different than downloading from Git?  This is the error when I try the changes you suggested:

CMake Error at library/src/CMakeLists.txt:37 (include):
  include could not find load file:

    /root/X/rocBLAS3-bs/build/release/virtualenv/cmake/TensileConfig.cmake


-- AMDGPU_TARGETS=gfx900
CMake Error at library/src/CMakeLists.txt:41 (TensileCreateLibrary):
  Unknown CMake command "TensileCreateLibrary".

Reply:
> I'm not familiar with virtualenv and my primitive attempts to implement the requested change were unsuccessful.

The virtualenv just allows for pip to install to a local directory instead of to the system. 

> Not sure what the -e switch does but the install location appears different than downloading from Git? This is the error when I try the changes you suggested:

This is a flag for pip that is documented [here](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs). From the docs, this does appear to install tensile with `python setup.py develop`(ie development mode) instead. From the [docs](https://setuptools.readthedocs.io/en/latest/setuptools.html#development-mode):

> To do this, use the setup.py develop command. It works very similarly to setup.py install or the EasyInstall tool, except that it doesn’t actually install anything. Instead, it creates a special .egg-link file in the deployment directory, that links to your project’s source code.

So this probably wont install the cmake file by default. Have you tried it without the `-e` flag?

I'll take a look at this in the morning as well.

Reply:
I checked this and it does work correctly just dropping the `-e` flag. So doing `virtualenv_install("-${Tensile_TEST_LOCAL_PATH}")` will install tensile from a local directory.